### PR TITLE
qTip2 v3.0.0 - Update info files

### PR DESCRIPTION
**Pull request by Craga89**
The `info.ini` and `update.json` files are now pointing to the wrong place, since the `qTip2/Bower` repository is gone in favour of a unified `qTip/qTip2 repository for both NPM and Bower packages.

### CODE REVIEW

File: files/qtip2/update.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "packageManager": "github",
+  "packageManager": "npm",
   "name": "qtip2",
-  "repo": "qtip2/bower",
+  "repo": "qtip2/qtip2",
   "files": {
-    "include": ["*.js", "*.css", "*.map", "./basic/**/*"]
+    "include": ["dist/**/*.js", "dist/**/*.css", "dist/**/*.map"]
```
Comment (jimaek) [inline files/qtip2/update.json]: If you want to include all files inside dist/ you can simply set `basePath` to `dist/` without any include parameters

**Comment by Craga89** [comment]
:+1: Updated to use basePath instead. Let me know if you'd like a rebase before the merge

File: files/qtip2/update.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
-  "packageManager": "github",
+  "packageManager": "npm",
   "name": "qtip2",
-  "repo": "qtip2/bower",
+  "repo": "qtip2/qtip2",
   "files": {
-    "include": ["*.js", "*.css", "*.map", "./basic/**/*"]
+    "basePath": "dist",
+    "include": ["**/*.js", "**/*.css", "**/*.map"]
```
Comment (jimaek) [inline files/qtip2/update.json]: remove the include line completely to add everything inside dist/ by default.
No need to rebase or anything

**Comment by Craga89** [comment]
:+1: Done, thanks @jimaek

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**